### PR TITLE
GitHub actions

### LINKED ISSUE #27: Implement GitHub Actions for Welcome Messages
When a PR or Issue is opened, let's have some GitHub actions do the magic by asking the users to do something or giving some useful information. Currently it's going to be experimental and we need to see the potential of how this can be used, but let's start with something small. 😊

### PR DESCRIPTION
**Pull request by mySTRiouscoder**
Github action to greet the contributors whenever they raise an issue or submit a PR.
Closes #27

### CODE REVIEW

File: .github/workflows/greetings.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Thank you for raising an issue.'
+        pr-message: 'Thank you for submitting a Pull Request'
```
Comment (praveenscience) [inline .github/workflows/greetings.yml]: Out of curiosity. Where do these messages appear? 😊 Can we also do something like this:
```suggestion
        issue-message: 'Thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
        pr-message: 'Thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```

Comment (mySTRiouscoder) [reply .github/workflows/greetings.yml]: @praveenscience  See this:

image

Comment (mySTRiouscoder) [reply .github/workflows/greetings.yml]: Also, we can surely do that. i'll edit it

Comment (praveenscience) [reply .github/workflows/greetings.yml]: @mySTRiouscoder Looks good. Can we also have @ mentions? 🤩

File: .github/workflows/greetings.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+name: Greetings
+
+on: [pull_request, issues]
+
+jobs:
+  greeting:
+    runs-on: ubuntu-latest
```
Comment (praveenscience) [inline .github/workflows/greetings.yml]: Well, why do we need `ubuntu-latest` here? Isn't this running on PRs and Issues? Just asking. If this needs to stay, we'll have it here, as is.

**Comment by mySTRiouscoder** [comment]
@praveenscience I edited it as you said , plus , i got to know about one more feature where-in, the bot can directly  mention the participant. Please see this:

image

**Comment by praveenscience** [comment]
Does it @ mention the user by default? Plus this is nice and a completely new issue we're talking here. Let me create a new issue for this.

File: .github/workflows/greetings.yml (hunk 2 above)
Comment (mySTRiouscoder) [reply .github/workflows/greetings.yml]: As far as i have read about it , it's like github actions provides some virtual environment(s) and the job(that we want the bot to perform) runs inside this virtual environment. 
So, yeah! This pretty much needs to stay.

**Comment by mySTRiouscoder** [comment]
> Does it @ mention the user by default? Plus this is nice and a completely new issue we're talking here. Let me create a new issue for this.

@praveenscience  Yes! It does it by default.

File: .github/workflows/greetings.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    steps:
+    - uses: actions/first-interaction@v1
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
+        issue-message: 'Hello @${{ github.actor }} ,Thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
+        pr-message: 'Hello @${{ github.actor }} ,Thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```
Comment (praveenscience) [inline .github/workflows/greetings.yml]: Sorry man, another suggestion. Please.
```suggestion
        issue-message: 'Hello @${{ github.actor }}, thank you for raising an issue. Please be detailed and clear along with an acceptance criteria.'
        pr-message: 'Hello @${{ github.actor }}, thank you for submitting a Pull Request. Make sure you have followed all the contribution guidelines.'
```

Comment (mySTRiouscoder) [reply .github/workflows/greetings.yml]: Updated @praveenscience . It's okay, no sorry ,instead points maybe? Haha !

**Comment by praveenscience** [comment]
Waiting for someone else to approve so this can be merged! 😊 If no one approves, I'll use my admin privileges! 😅

**Comment by mySTRiouscoder** [comment]
@vermakhushboo I made the changes.

**Pull request merged**